tests/child_thread/host/host.cpp

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/host.h>
+#include <openenclave/internal/calls.h>
+#include <openenclave/internal/error.h>
+#include <openenclave/internal/tests.h>
+#include <chrono>
+#include <cstdio>
+#include <cstdlib>
+#include <thread>
+
+#include "child_thread_u.h"
+
+#define ECALL_IN_CHILD_THREAD 0
+#define DESTROY_IN_CHILD_THREAD 1
+#define DESTROY_AND_CHECK_CHILD_THREAD 2
+
+bool multi_thread_flag = true;
+
+void stay_ocall(void)
+{
+    // sleep for 5 seconds.
+    std::this_thread::sleep_for(std::chrono::milliseconds(5000));
```

Comment:
If we really want to be sure that this thread will stay in ocall, sleeping for 5 seconds may not be enough since the thread scheduler could suspend other threads too for a long time if the machine is under a heavy load.

 It would be more robust to use a std::condition_variable instead so that the main thread can precisely wake this thread when it wants to.